src/DssDirectDepositHub.sol

```diff
@@ -442,10 +441,12 @@ contract DssDirectDepositHub {
                     currentAssets
                 );
             } else {
-                // All the subtractions are safe as otherwise toUnwind is > 0
```

Comment:
why it was removed?

Reply:
i think i removed it before `unchecked` was added so safemath was in place.  adding it back